src/Features/LanguageServer/ProtocolUnitTests/Ordering/RequestOrderingTests.cs

```diff
@@ -246,15 +246,22 @@ private static async Task ExecuteDidOpen(TestLspServer testLspServer, Uri docume
                     Text = "// hi there"
                 }
             };
-            await testLspServer.ExecuteRequestAsync<LSP.DidOpenTextDocumentParams, object>(Methods.TextDocumentDidOpenName, didOpenParams, new LSP.ClientCapabilities(), null, CancellationToken.None);
+            await testLspServer.ExecuteRequestAsync<LSP.DidOpenTextDocumentParams, object>(Methods.TextDocumentDidOpenName, didOpenParams, CancellationToken.None);
         }
 
-        private static async Task<Solution> GetLSPSolution(TestLspServer testLspServer, string methodName)
+        private static async Task<Solution?> GetLSPSolution(TestLspServer testLspServer, string methodName)
         {
             var request = new TestRequest(methodName);
-            var response = await testLspServer.ExecuteRequestAsync<TestRequest, TestResponse>(request.MethodName, request, new LSP.ClientCapabilities(), null, CancellationToken.None);
+            var response = await testLspServer.ExecuteRequestAsync<TestRequest, TestResponse>(request.MethodName, request, CancellationToken.None);
             Contract.ThrowIfNull(response);
-            return response.Solution;
+            if (response.ContextHasSolution)
```

Comment:
similar to before, solutions from the test workspace are not easily serializable.  Instead we verify that we get thecorrect lsp solution from the manager.